fixtures/small.yaml

```diff
@@ -0,0 +1,159 @@
+  fields:
+    password: pbkdf2_sha256$180000$hxOg7JZvMUQG$lJ2dnEhPILwtVr9EMFHl4RPPx+1FQltJ8Xp7fTsH0fk=
+    last_login:
+    is_superuser: false
+    username: helper1
+    first_name: Helper
+    last_name: Jonh
+    is_staff: false
+    is_active: true
+    date_joined: '2020-04-04T18:00:32Z'
+    email: helper1@email.se
+    groups: []
+    user_permissions: []
+- model: authentication.user
+  pk: 3
+  fields:
+    password: pbkdf2_sha256$180000$pwOJIR4J6zjE$ZkYFrFO2i4ZdOfhXm5pwBKIqyellJSWvDSxyqVgFDPQ=
+    last_login:
+    is_superuser: false
+    username: helper2
+    first_name: Helper
+    last_name: Joe
+    is_staff: false
+    is_active: true
+    date_joined: '2020-04-04T18:01:31Z'
+    email: helper2@email.se
+    groups: []
+    user_permissions: []
+- model: authentication.user
+  pk: 4
+  fields:
+    password: pbkdf2_sha256$180000$J8vKz70ElQHJ$hGUY07FWga78SeX6syZ6+LxG+hrUbleeqh408/OEHIo=
+    last_login:
+    is_superuser: false
+    username: affected1
+    first_name: Marc
+    last_name: Affected
+    is_staff: false
+    is_active: true
+    date_joined: '2020-04-04T18:06:39Z'
+    email: affected1@email.se
+    groups: []
+    user_permissions: []
+- model: authentication.user
+  pk: 5
+  fields:
+    password: pbkdf2_sha256$180000$dxZbeCG58gQf$nNehCSJIyiZjek2JFVXxAHhgidDkj2ybYvLSpqc4ZkU=
+    last_login:
+    is_superuser: false
+    username: affected2
+    first_name: Mary
+    last_name: Affected
+    is_staff: false
+    is_active: true
+    date_joined: '2020-04-04T18:10:45Z'
+    email: affected2@email.se
+    groups: []
+    user_permissions: []
+- model: management.participant
+  pk: 1
+  fields:
+    deleted:
+    user: 2
+    type: HL
+    crisis: 1
+    first_line_of_address: Vasagatan, 46
+    second_line_of_address: "-"
+    country: SE
+    place_id: 83M4+HJ Stockholm
+    position:
+    post_code: '11120'
+    city: Stockholm
+    phone: '12341234'
+    metadata: {}
+    created_at: '2020-04-04T18:04:33.486Z'
+- model: management.participant
+  pk: 2
+  fields:
+    deleted:
+    user: 3
+    type: HL
+    crisis: 1
+    first_line_of_address: Rådmansgatan 70A
+    second_line_of_address: s
+    country: SE
+    place_id: 83Q3+PJ Stockholm
+    position:
+    post_code: '11360'
+    city: Stockholm
+    phone: '12341234'
+    metadata: {}
+    created_at: '2020-04-04T18:06:18.414Z'
+- model: management.participant
+  pk: 3
+  fields:
+    deleted:
+    user: 4
+    type: AF
+    crisis: 1
+    first_line_of_address: Karlavägen 81
+    second_line_of_address: "-"
+    country: SE
+    place_id: 83QP+CR Stockholm
+    position:
+    post_code: '11459'
+    city: Stockholm
+    phone: '12341234'
+    metadata: {}
+    created_at: '2020-04-04T18:08:47.811Z'
+- model: management.participant
+  pk: 4
+  fields:
+    deleted:
+    user: 5
+    type: AF
+    crisis: 1
+    first_line_of_address: Västerlånggatan 48
+    second_line_of_address: "-"
+    country: SE
+    place_id: 83FC+J6 Stockholm
+    position:
```

Comment:
Oo, will this position go through ? I thought we had to have a geo point here ?

Reply:
Without this, I get a feeling we might end up with an exception on calling v1/affected-participants/ ? Should we fix it ?

Reply:
I can see how a point field looks on the JSON though. https://stackoverflow.com/a/59539162 dont know how helpful that is.

Reply:
Cool, I didn't know what to provied, what about the place_id, is that correct? I also didn't know what to provide

Reply:
So you can see an example placeId on the API docs herehttp://docs.quarantinehelp.space/#/Authentication/post_api_v1_auth_register_   `"placeId": "ChIJwyyKo7J3X0YRZ5XOMcLx3xo"` It is more like a HERE maps placeId

Reply:
@tonythomas01 pushed adjustments, can you please take a look again?